tests/org.obeonetwork.m2doc.tests/src/org/obeonetwork/m2doc/test/AbstractM2DocTest.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ *  Copyright (c) 2016 Obeo. 
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *   
+ *   Contributors:
+ *       Obeo - initial API and implementation
+ *  
+ *******************************************************************************/
+package org.obeonetwork.m2doc.test;
+
+import java.util.Collection;
+
+import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.emf.common.util.URI;
+import org.eclipse.emf.ecore.resource.Resource;
+import org.eclipse.sirius.business.api.session.SessionManager;
+import org.eclipse.sirius.tests.support.api.EclipseTestsSupportHelper;
+import org.eclipse.sirius.tests.support.api.SiriusDiagramTestCase;
+import org.junit.Before;
+
+/**
+ * @author pguilet<pierre.guilet@obeo.fr>
+ */
+public abstract class AbstractM2DocTest extends SiriusDiagramTestCase {

```

Comment:
This cool. Just, I don't get why AbstractM2DocTest extends SiriusDiagramTestCase. May be the name isn't well suited.